src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -274,18 +274,32 @@ String membersOfConversation = (String) request.getAttribute("membersOfConversat
 
       <div id="chatBox" class="rounded">
         <div id="messagesContainer">
-          <ul>
-            <%
-              for (Message message : messages) {
-                String author = UserStore.getInstance().getUser(message.getAuthorId()).getName();
-            %>
-                <li> <strong> <a href="/profile/<%=author %>"><%= author %></a>: </strong>
-                <%= message.getContent() %> </li>
-
-            <%
-                }
-            %>
-          </ul>
+          <%
+            for (Message message : messages) {
+              String author = UserStore.getInstance().getUser(message.getAuthorId()).getName();
+              if(user.equals(author)){
+          %>
+                <div class="messageContainer right">
+                  <div class="box rightBubble blue">
+                    <%= message.getContent() %>
+                  </div>
+                  <img class="dot rightImg" src="/default-user.png">
+                </div>
+          <%
+              }else{
+          %>
+                <div class="messageContainer">
+                  <img class="dot leftImg" src="/default-user.png">
+                  <div class="box leftBubble">
+                    <a class="author" href="/profile/<%=author%>"> <%=author%> </a> 
```

Comment:
maybe we can add a link on the image of the right hand side user

Reply:
Do you mean clicking on the image could redirect to the logged in user's profile?